content/cloud/develop.md

```diff
@@ -205,13 +205,74 @@ $ spin plugins install js2wasm --yes
 
 {{ startTab "Python" }}
 
-You'll need the Spin `py2wasm` plugin:
+You may have see a `py2wasm` plugin in your travels. Please note, `py2wasm` (an experiment to build a Spin Python SDK using CPython, Wizer, and PyO3) has since been replaced by Componentize-Py. The process of getting your system ready to write Wasm-powered Python applications is as follows:
 
 <!-- @selectiveCpy -->
 
 ```bash
-$ spin plugins update
-$ spin plugins install py2wasm --yes
+# As shown above, we install the Python SDK (which provides us with Spin's http-py tempate)
+$ spin templates install --git https://github.com/fermyon/spin-python-sdk --update
+```
+
+Once we have the above `spin-python-sdk` installed we can scaffold out a new App using the `http-py` template. The scaffolded App has a `requirements.txt` file that facilitates the installation of `spin-sdk` and `componentize-py`. While you could manually install these using Pip, the `requirements.txt` file has the appropriate version numbers set making the process quicker and also more robust. Let's create a new Spin app and run harness the `requirements.txt`:
```

Comment:
This doesn't install the `spin-python-sdk` - it installs the Spin template(s) _from the `spin-python-sdk` repository_.

Reply:
Thanks @itowlson.